collection-scripts/gather_etcd

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+ETCD_LOG_PATH="${BASE_COLLECTION_PATH}/etcd_info"
+
+ETCDCTL_CONTAINER='etcdctl'
+
+RUNNING_ETCD_POD=$(oc get pod -n openshift-etcd --no-headers|grep Running| grep -o -m 1 '\S*etcd\S*')
+
+# We cannot rely on ETCDCTL_ENDPOINTS on container because it may contain bootstrap VM
+ETCDCTL_ENDPOINTS=$(oc exec ${RUNNING_ETCD_POD}  -n openshift-etcd -c ${ETCDCTL_CONTAINER} -- /bin/sh -c "etcdctl member list" | awk -F', ' '{printf "%s%s",sep,$5; sep=","}')
+
+function ocp4etcdctl {
+    oc -n openshift-etcd exec -c ${ETCDCTL_CONTAINER} ${RUNNING_ETCD_POD} -- env ETCDCTL_ENDPOINTS=${ETCDCTL_ENDPOINTS} etcdctl $@
+}
+
+if [[ -z $RUNNING_ETCD_POD ]];then
+    echo "ERROR: No running etcd pods found" >&2
+    exit 1
+fi
+
+echo "Getting information from pod \"${RUNNING_ETCD_POD}\", container \"${ETCDCTL_CONTAINER}\""
+echo "Using endpoints: ${ETCDCTL_ENDPOINTS}"
+
+mkdir -p ${ETCD_LOG_PATH}
+
+echo "INFO: Starting getting etcd information"
+
+PIDS=()
+
+# member list
+echo "INFO: Getting etcdctl member list"
+ocp4etcdctl member list -w table > ${ETCD_LOG_PATH}/member_list.txt &
```

Comment:
Table output is convenient, but the JSON output should be sufficient, right?  I'd rather not bloat the must-gathers with redundant information, since folks can always write consuming code that pretty-prints the JSON if they want convenient inspection.

Reply:
Right: I pasted both JSON and table outputs because one of the most challenging aspects of dealing with must-gather is having all the information in machine-readable format only. However, I understand the need of no redundancy, so let me fix it.